templates/4a_compensate_image_data.ipynb

```diff
@@ -38,9 +38,8 @@
     "import shutil\n",
```

Comment:
<p>I would clarify that cohort name is the name of the folder that will store the rosetta testing files, not that it's the final name of their cohort</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>